spec/requests/posts_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      user = User.create(name: 'jane', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Benin.',
+                         posts_counter: 0)
+      get user_posts_path(user)
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      user = User.create(name: 'jane', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Benin.',
+                         posts_counter: 0)
+      get user_posts_path(user)
+      expect(response).to render_template(:index)
+    end
```

Comment:
- [ ] According to the requirements:

![Screenshot from 2023-08-02 09-29-25](https://github.com/rugwizangoga/Blog-app/assets/77069011/5635fc77-e16a-4a0a-9c88-f08ccbe52e7f)

This test suite is missing the test of the `response.body`. Please make sure to include it! 🤓 🔧